tools/SetupFlow/DevHome.SetupFlow.UnitTest/TestingScenarios/Quickstart.md

```diff
@@ -0,0 +1,36 @@
+# Quickstart Playground Tests
+If your code affects Quickstart Playground, please manually verify these scenarios.
+
+## Scenarios
+Please make sure to verify all of these scenarios. These apply to both Windows 10 and Windows 11.
+
+### First Run Experience (Machine Has Never Run Quickstart Playground Before)
```

Comment:
Is uninstalling and reinstalling Dev Home enough to make Quickstart think this is the first run?

Reply:
Oh, hm, I guess maybe First Run is misleading here and I should fix that? This test scenario is trying to verify the process for configuring dependencies too (in addition to turning on the experimental feature for the first time)- so we'd ideally want the machine to not have things like the API key already configured or some subset of the app dependencies already installed (the feature depends on Visual Studio Code, Docker Desktop, and the Remote Containers extension for VS Code being present). We have some adaptive cards to help configure those things if they are not already present, which is what we want to validate in the test.

Reply:
I rewrote this to hopefully better capture the intent. Let me know if you'd suggest anything else here.